packages/select/src/vaadin-select-overlay.js

```diff
@@ -5,18 +5,20 @@
  */
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { defineCustomElement } from '@vaadin/component-base/src/define.js';
-import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
-import { OverlayMixin } from '@vaadin/overlay/src/vaadin-overlay-mixin.js';
-import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
 import { overlayStyles } from '@vaadin/overlay/src/vaadin-overlay-styles.js';
 import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { SelectOverlayMixin } from './vaadin-select-overlay-mixin.js';
 
 const selectOverlayStyles = css`
   :host {
     align-items: flex-start;
     justify-content: flex-start;
   }
 
+  [part='overlay'] {
+    min-width: var(--vaadin-select-overlay-width, var(--vaadin-select-text-field-width));
+  }
```

Comment:
This should be scoped to exclude fullscreen overlay:

![Screenshot 2024-07-24 at 13 25 40](https://github.com/user-attachments/assets/161b2e51-5247-421f-947b-a5c2f023ae5a)

Reply:
Thanks, fixed.